modules/schema/src/test/java/org/apache/ignite/internal/schema/serializer/AbstractSerializerTest.java

```diff
@@ -419,4 +376,46 @@ public String toString() {
             return type.spec().name() + ": " + AbstractSerializerTest.toString(defaultValue);
         }
     }
+
+    /** Creates a native type from given type spec. */
```

Comment:
can we have unified single place for all such places?
I see at least 4 points: itColocationTest,ProjectedTupleTest,AbstractSerializerTest

Reply:
@ygerzhedovich done